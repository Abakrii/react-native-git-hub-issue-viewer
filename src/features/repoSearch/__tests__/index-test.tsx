import React from 'react';
import renderer from 'react-test-renderer';
import RepoSearch from '../';

test('renders correctly', () => {
  const tree = renderer.create(<RepoSearch />).toJSON();
  expect(tree).toMatchSnapshot();
});
