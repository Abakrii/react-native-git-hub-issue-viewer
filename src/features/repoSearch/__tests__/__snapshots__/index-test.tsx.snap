// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#FFF",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "padding": 40,
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      placeholder="Paste a link to a GitHub repo!"
      rejectResponderTermination={true}
      style={
        Object {
          "borderBottomColor": "#DDD",
          "borderBottomWidth": 1,
          "marginBottom": 10,
          "paddingHorizontal": 8,
          "paddingVertical": 6,
        }
      }
      underlineColorAndroid="transparent"
    />
    <View
      accessibilityRole="button"
      accessibilityState={
        Object {
          "disabled": true,
        }
      }
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {},
            Object {},
          ]
        }
      >
        <Text
          disabled={true}
          style={
            Array [
              Object {
                "color": "#007AFF",
                "fontSize": 18,
                "margin": 8,
                "textAlign": "center",
              },
              Object {
                "color": "coral",
              },
              Object {
                "color": "#cdcdcd",
              },
            ]
          }
        >
          Get repo issue
        </Text>
      </View>
    </View>
    <Text
      style={
        Object {
          "color": "red",
          "fontSize": 20,
          "fontWeight": "bold",
        }
      }
    />
  </View>
</View>
`;
